server/services/deployment.spec.ts

```diff
@@ -51,4 +61,42 @@ describe('test deployment service', () => {
     expect(deployment._id).not.toBe(undefined)
     expect(deployment.uuid).toBe(testDeployment2.uuid)
   })
+
+  test('that we can email deployment owners about changes for multiple deployments', async () => {
+    const deployments: DeploymentDoc[] = [testDeployment, testDeployment2]
+    const spy: any = jest.spyOn(entityUtils, 'getUserListFromEntityList')
+    const userList: object[] = [
+      { name: 'Alice', email: 'alice@email.com' },
+      { name: 'Bob', email: 'bob@email.com' },
+    ]
+    spy.mockReturnValue(userList)
+
+    await emailDeploymentOwnersOnVersionDeletion(deployments, testVersion)
+
+    expect(emailClient.sendEmail).toHaveBeenCalledTimes(deployments.length * userList.length)
+  })
+
+  test('that we can email deployment owners about changes for a single deployment', async () => {
+    const deployments: DeploymentDoc[] = [testDeployment]
+    const spy: any = jest.spyOn(entityUtils, 'getUserListFromEntityList')
+    const userList: Array<any> = [
```

Comment:
Instead of making new test data here you could construct the array of users by using the existing test data in `utils/test/testModels`. E.g. `testUser` and `testUser2`.